setup.py

```diff
@@ -80,6 +84,9 @@
     if arg == "--no-ninja":
         NO_NINJA = True
         continue
+    if arg == "--build-with-ucc":
```

Comment:
I think we also needed a cmake_prefix_path to actually find where the UCC libraries are? not sure if that part can be done automatically. i.e. do they have some python module/utils to query where the library is?